packages/react-workspace/agents.md

```diff
@@ -0,0 +1,19 @@
+# Nimble React – AI Instructions
+
+## Overview
+React wrappers for Nimble Web Components.
+- **Status**: Alpha / Experimental
+- **Pattern**: `useLayoutEffect` for events, `forwardRef` for DOM access.
+
+## Key References
+- `README.md` – Current status.
+
+## Core Patterns
+- Ensure custom elements are registered before use.
+- Use `useLayoutEffect` to attach event listeners to custom events.
+- Forward refs to the underlying DOM element.
+
+## Common Pitfalls
+- ❌ **`useEffect` for Events**: Use `useLayoutEffect` to avoid timing issues with custom elements.
+- ❌ **Missing Ref Forwarding**: Wrappers must forward refs to the underlying web component.
```

Comment:
I don't know what this means but no, our React wrappers don't forward ref explicitly. It's handled by our wrap helper.